docs/src/Guides/23 Voice.md

```diff
@@ -102,13 +102,14 @@ import interactions
 
 @interactions.slash_command("record", "record some audio")
 async def record(ctx: interactions.SlashContext):
+    await ctx.defer()
     voice_state = await ctx.author.voice.channel.connect()
 
     # Start recording
     await voice_state.start_recording()
     await asyncio.sleep(10)
     await voice_state.stop_recording()
-    await ctx.send(files=[interactions.File(file, file_name="user_id.mp3") for user_id, file in voice_state.recorder.output.items()])
+    await ctx.send("Here are your recordings", files=[interactions.File(file, file_name="user_id.mp3") for user_id, file in voice_state.recorder.output.items()])
```

Comment:
The content here shouldn't be needed. What error are you getting here?

Reply:
I cannot test again until a bit later, but there was an error about the message not being empty.

Reply:
It's because you didn't actually send/capture any audio. You need to talk and should visually see that your microphone is activating. Without any text or file attachments, the message was empty, hence the error.

```python
files=[interactions.File(file, file_name=f"{user_id}.mp3") for user_id, file in voice_state.recorder.output.items()]
if len(files) > 0:
    await ctx.send(files=files)
else:
    await ctx.send("No audio was recorded.")
```